main.rb

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+
+require_relative 'app'
+
+app = App.new
+app.run
```

Comment:
* According to the project requirement your `main.rb` method should do the following 

> Present the user with a list of options to perform.
> Let users choose an option.
> If needed, ask for parameters for the option.
> Have a way to quit the app
* Kindly implement all these 🙏 **HINT** write a loop that displays all available options to users, then write conditions to call a specific method depending on the users' choice